lib/configuru/config_methods.rb

```diff
@@ -11,37 +11,38 @@ module ClassMethods
       def param_names
         @param_names ||= []
       end
-      def param(name,options={})
+
+      def param(name, options={})
```

Comment:
Assignment Branch Condition size for param is too high. [88.61/15]<br>Surrounding space missing in default value assignment.